backend/test/test_org.py

```diff
@@ -555,3 +559,147 @@ def test_update_read_only(admin_auth_headers, default_org_id):
     assert data["readOnly"] is False
     # Test that reason is unset when readOnly is set to false, even implicitly
     assert data["readOnlyReason"] == ""
+
+
+def test_create_org_and_invite_new_user(admin_auth_headers):
+    invite_email = "new-user@example.com"
+    r = requests.post(
+        f"{API_PREFIX}/orgs/create",
+        headers=admin_auth_headers,
+        json={
+            "firstAdminInviteEmail": invite_email,
+            "quotas": {
+                "maxPagesPerCrawl": 100,
+                "maxConcurrentCrawls": 1,
+                "storageQuota": 1000000,
+                "maxExecMinutesPerMonth": 1000,
+            },
+            "subData": {"extra": "data", "sub": {"id": 123}},
+        },
+    )
+
+    assert r.status_code == 200
+    data = r.json()
+    assert data["added"]
+
+    org_id = data["id"]
```

Comment:
Maybe worth testing that the new org's name is a string representation of its id at this point?

Reply:
Will update in follow-up PR, which adds additional tests.